endoreg_db/management/commands/import_video.py

```
@@ -204,38 +195,32 @@ def handle(self, *args, **options):
+                raise AssertionError(
+                    f"No processors linked to '{center.name}' and no processor called '{processor_name}' exists. "
+                    "Fallback from default Processor applied."
                 )
-                return
-
             elif proc_count == 1:
                 processor = processors_qs.first()
             else:
+
                 processor = self._choose_processor_interactively(processors_qs)
+                self.stdout.write(self.style.SUCCESS(f"Using processor: {processor.name}"))
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "raise", remove the leading "el" from "elif"

(R1720)